packages/rich-text/src/plugins/Hyperlink/components/UrlHyperlink.tsx

```diff
@@ -31,36 +33,33 @@ type HyperlinkElementProps = {
 export function UrlHyperlink(props: HyperlinkElementProps) {
   const editor = useContentfulEditor();
   const sdk: FieldAppSDK = useSdkContext();
-  const { uri } = props.element.data;
+  const focus = editor.selection?.focus;
```

Comment:
Maybe not for this MR, but we have some duplication in the Hyperlink components, maybe we can have a base component/hook to share the logic and only provide the specifics to them

Reply:
I'll create a ticket so we can work on that (maybe during cooldown)